### PR TITLE
Lingujiang

### PR DESCRIPTION
**Pull request by lingujiang**
1.add input_p.txt for tests questions
2.add parse_question.py as a script to generate tests questions json file
3.generate tests questions json file

### CODE REVIEW

File: IronMan_MK1/tests/data/parse_question.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
+import json
+import re
+
+def writeJosn(filename,data):
+    file_full_name = './'+filename+'.json'
+    with open(file_full_name,'w') as file_out:
+        json.dump(data, file_out,indent=4)
+
+
```
Comment (lomo44) [inline IronMan_MK1/tests/data/parse_question.py]: - All of the code below should be inside an if __name__ == "__main__" block

Comment (lomo44) [inline IronMan_MK1/tests/data/parse_question.py]: - the code below should be inside an if __name__ == "__main__" block

File: IronMan_MK1/tests/data/parse_question.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
+import json
+import re
+
+def writeJosn(filename,data):
```
Comment (lomo44) [inline IronMan_MK1/tests/data/parse_question.py]: - typo?

File: IronMan_MK1/tests/data/input_p.txt [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Answer: Blue
+Question Closed 4: What's your hair color?
+Answer: Black
+Question Closed 5: Where are you living now?
+Answer:  New York
+Question Closed 6: Where did you go for your undergraduate?
+Answer: MIT
+Question Closed 7: What's major in university?
+Answer: Machnical Engineering and Electrical Engineering
+Question Closed8: Who is your father?
```
Comment (lomo44) [inline IronMan_MK1/tests/data/input_p.txt]: - inconsistent format

File: IronMan_MK1/tests/data/parse_question.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        if match_answer:
+            question[1] = match_answer.group(1)
+            if question_type == 'Polar':
+                list_element = {'Question': question[0], 'Answer': question[1]}
+                q_polar.append(list_element)
+            if question_type == 'Closed':
+                list_element = {'Question': question[0], 'Answer': question[1]}
+                q_closed.append(list_element)
+    data = {'Polar Question':q_polar,'Closed Question':q_closed,'Open_ended Question':q_open}
+
```
Comment (lomo44) [inline IronMan_MK1/tests/data/parse_question.py]: - should we considered multiple possible answer in the future?

File: IronMan_MK1/tests/data/parse_question.py (hunk 1 above)
Comment (lingujiang) [reply IronMan_MK1/tests/data/parse_question.py]: fixed

File: IronMan_MK1/tests/data/parse_question.py (hunk 2 above)
Comment (lingujiang) [reply IronMan_MK1/tests/data/parse_question.py]: fixed

File: IronMan_MK1/tests/data/input_p.txt (hunk 3 above)
Comment (lingujiang) [reply IronMan_MK1/tests/data/input_p.txt]: fixed

File: IronMan_MK1/tests/data/parse_question.py (hunk 4 above)
Comment (lingujiang) [reply IronMan_MK1/tests/data/parse_question.py]: I think for polar and closed question, there will be only one answer?

**Pull request merged**